sst/weatherForecast/Weather_Forecast.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    from lsst_efd_client import EfdClient
</pre><p>Consider using the following import and functions instead:</p>

<pre class="ql-syntax" spellcheck="false">from lsst.summit.utils.efdUtils import getEfdData, makeEfdClient
</pre><p>The advantage of this version is that it uses sync functions instead of async functions. It also selects the EFD version depending if your notebook is running on the Summit or on USDF. I recommend you start using this in all your notebooks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done, I added in a new cell. </p>